py3status/modules/mpd_status.py

```diff
@@ -6,6 +6,10 @@
     cache_timeout: how often we refresh this module in seconds (default 2)
     format: template string (see below)
         (default '%state% [[[%artist%] - %title%]|[%file%]]')
+    format_mpd_not_authenticating: string to print
+        (default 'mpd_status: Authentication failed')
+    format_mpd_not_running: string to print
+        (default 'mpd_status: Connection refused')
```

Comment:
> I like the feeling of making a string customizable by start typing off with format_. Even for icons too. If we change format_clone to icon_clone, then why not separator_clone or symbol_clone?

from my point of view format_... uses the formatter so I can do `'[\?color=bad ...]'` etc but the icon_... does not do this sort of thing so has a different name style, since this happens often then we should have a convention and some modules had icon_... so that has become the convention

the whole error stuff has no convention at present as it is all hard coded.  notification_... is bad imho as notifications have meaning eg they are notify-send/i3-nagbar type stuff via py3.notify_user() etc